<?xml version="1.0"?>
<launch>

  <!-- <node pkg="zed_publisher" type="decompress" name="decompress" args="manager" /> -->

  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <!-- <remap from="camera_info" to="/zed2/zed_node/rgb/camera_info"/> -->
    <remap from="camera_info" to="/cam_info"/>
    <remap from="image_rect" to="/my_depth"/>
    <param name="queue_size" type="int" value="100"/>
  </node>

  <!--node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
    <remap from="rgb/camera_info" to="/zed2/zed_node/rgb/camera_info"/>
    <remap from="rgb/image_rect_color" to="/zed2/zed_node/rgb/image_rect_color"/>
    <remap from="depth_registered/image_rect" to="/zed2/zed_node/depth/depth_registered"/>
    <remap from="depth_registered/points" to="/point_cloud"/>
  </node-->

</launch>